client/core/trade.go

```diff
@@ -318,12 +323,13 @@ func (t *trackedTrade) readConnectMatches(msgMatches []*msgjson.Match) {
 		corder, _ := t.coreOrderInternal()
 		t.notify(newOrderNote("Missing matches", details, db.ErrorLevel, corder))
 		for _, mid := range missing {
-			log.Errorf("%s did not report active match %s on order %s", host, mid, t.ID())
+			log.Warnf("%s did not report active match %s on order %s", host, mid, t.ID())
 		}
 	}
 	if len(extras) > 0 {
 		details := fmt.Sprintf("%d matches reported by %s were not found for %s.", len(extras), host, t.token())
 		t.notify(newOrderNote("Match resolution error", details, db.ErrorLevel, corder))
+		// t.negotiate(extras) // missed match message request, but match not revoked (?!)
```

Comment:
Just a thought.  If the match isn't revoked yet, the pending match request should be coming, so this should not be necessary though.